libs/ngx-nuts-and-bolts/src/lib/testing/README.md

```diff
@@ -0,0 +1,13 @@
+# ExtractPublic
+
+Use ExtractPublic custom type when you want to extract public members of class. It could prove most useful when creating test doubles for your services or components[^1].
```

Comment:
```suggestion
Use `ExtractPublic` custom type when you want to extract public members of class. It could prove most useful when creating test doubles for your services or components[^1].
```